src/App/App.js

```diff
@@ -14,10 +14,36 @@ class App extends Component {
         console.log(this.state.movies)
     }
 
+    componentDidMount = () => {
+        fetch("https://rancid-tomatillos.herokuapp.com/api/v2//movies/")
+        .then(res => {
+           if(!res.ok) {
+               throw new Error('Failed to fetch movies')
+           }
+           return res.json()})
+        .then(moviesData => this.setState({
```

Comment:
As you've gotten your fetches set up, you'll want to remove the hard coded movieData from your app altogether.  Think about what makes sense to use as your initial value of the movies property.   What can give other devs quick insight into what shape that data will eventually take?